### PR TITLE
v1.13.0

### PR DESCRIPTION
**Pull request by ahuang11**
geoviews 1.13.0

Destination channel: defaults

Links
[Upstream repository](https://github.com/holoviz/geoviews)
[Upstream changelog/diff](https://github.com/holoviz/geoviews/compare/v1.12.0...v1.13.0)
Explanation of changes:
https://github.com/conda-forge/geoviews-feedstock/pull/41/files

### CODE REVIEW

**Comment by ahuang11** [comment]
Supersedes https://github.com/AnacondaRecipes/geoviews-feedstock/pull/13 now that I have permission to make branches directly in this repo.

**Comment by ahuang11** [comment]
Not sure why the tests are failing

File: recipe/meta.yaml
Comment (ahuang11) [inline recipe/meta.yaml]:
```suggestion
        - bokeh =3.5
```

Comment (philippjfr) [reply recipe/meta.yaml]: Should probably be `bokeh >=3.5,<3.7` since we already determined that Panel>=1.5.1 has full compatibility with both Bokeh 3.5.x and 3.6.x

File: recipe/meta.yaml
Comment (ahuang11) [inline recipe/meta.yaml]:
```suggestion
        - bokeh >=3.5,<3.7
```

**Comment by maximlt** [comment]
@ahuang11 do you think this is unblocked now Panel 1.5.2 is released?

**Comment by ahuang11** [comment]
@maximlt can you provide a final review before I merge?

**Review by maximlt — APPROVED** [review]

**Comment by maximlt** [comment]
@ahuang11 as indicated in our confluence doc, you need to ask a review from the distro team on Slack at `#package-review-team`. My review isn't worth much here, usually I see at least two reviews from this team.

Then, I usually let them merge the PR, but IIRC I have been told I could merge it too once approved by them. Somehow I prefer when they merge as anyway merging doesn't release the package yet, someone still needs to perform another step pushing it to the channel. I kind of assume the person merging is also going to be the person pushing the package, I may be wrong though.

**Comment by cbouss** [comment]
Upload is performed by package builder in a separate process. If you merge yourself we may miss the upload step.

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
        - matplotlib-base >2.2
```
A lightweight part of the `matplotlib` package (without `pyqt`)

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml]: @ahuang11 Could you revert this change? Platforms will fail on our CI (Prefect) randomly if `meta.yaml` doesn't have a generic `requirements/host` section

Comment (cbouss) [reply recipe/meta.yaml]: Is there a conda-build issue filed for this? Is it because of skip py on the global level without python being on the global env?

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**
